docs/design/alias-configuration.md

```diff
@@ -52,4 +68,55 @@ In this flow
+  - Recovered address must match with the alias (public address of the eth account)
+  - If the recovered address match with the address stored in the alias we have to update the key property of the account and store it in the ledger
+  - With the above step the process of lazy creation will be finished
+- Crypto Transfer Transaction to ECDSA key alias should create an account with its alias field set to the ECDSA key, populate the alias map with the EVM address calculated based on the ECDSA key, and the key field set with the value of the ECDSA key (currently implemented) and another Crypto Transfer Transaction to the EVM address corresponding to the ECDSA key should transfer to the account created in the previous transaction
+- Crypto Create Transaction with ECDSA key alias:
+    
+**Always validate the 0.0.ECDSA alias format is missing from the alias map as well as the calculated evm_address alias format**
+    
+  1. Create an account with its alias and key fields set to the ECDSA key, and populate the alias map with the EVM address calculated based on the ECDSA key.
+      - Account with:
+          - hedera id: 0.0.20
+          - admin key: ECDSA key
+          - alias: 0.0.ECDSA key
+      - AliasManager map with entries:
+          - calculated EVM_address alias based on the ECDSA key → hedera id
+          - 0.0.ECDSA key → hedera id
+  
+  2. Attempting another Crypto Create Transaction with the same ECDSA key alias or EVM address corresponding to that key should fail
+    
+- Crypto Create Transaction with EVM address alias
+    1. Create an account with its alias field set to the EVM address and its key will be empty
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key:
+            - alias: EVM_address alias
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+    2. a) When a signed transaction referencing the account by the EVM_address alias is received:
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key: ECDSA key
+            - alias: 0.0.ECDSA key
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+            - map with the 0.0.ECDSA key alias → hedera id
+    
+    2. b) Don’t overwrite the alias field, instead add logic during the map rebuilding
+    - if the value of the alias field of the account has the evm_address format, add an entry to the aliases map with the value of 0.0.ECDSA key which is set on the account
+
+### Notes
+
+- For the EVM address cases we should store the alias value in the state not only in the memory structure
+- It is ok for different accounts to have the same key
+- For accounts with non-empty aliases if we have a key that is set, it should match the alias (for key aliases the values should be equal and for EVM aliases the key should be the ECDSA key from which the EVM address is derived)
```

Comment:
Can you add a test case and design logic for 
`EthereumTransaction` where the t`x.value` is set (and therefore it's a cryptotransfer) but the ethereum adddress noted in `tx.to` is not. 
We should do a lazy create in this case and create a hollow account

Reply:
@Nana-EC I'm not sure I understand how this will work. We need at least an `evm_address` alias for the lazy created account. What would the alias be in this case, if `to` is not set in the `EthereumTransaction`?

Reply:
This would only refer to the case were `to` is set and `value` is non zero.
It's basically to expose account creation that we do via HAPI when sending HBAR via the EthereumTransaction to support Ethereum RPC native wallets and tools

Reply:
Added additional case.